docs/languageversion2.md

```diff
@@ -0,0 +1,11 @@
+# Language Version 2.0 Triggers
+
+We have recently developed a new Language Version 2.0, which provides a new format for resource definitions when transpiling to an ARM Template.
```

Comment:
We should link to the docs on this: https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/syntax#languageversion-20